apps/docs/pages/guides/platform/database-size.mdx

```diff
@@ -71,6 +71,12 @@ Due to provider restrictions, disk expansions can occur only once every six hour
 
 </Admonition>
 
+<Admonition type="info">
```

Comment:
Looks fine, though perhaps we should also add a note to the caveats section of the upgrades doc?